generators/entity/templates/server/src/main/java/package/common/field_validators.ejs

```diff
@@ -51,14 +51,15 @@ if (field.fieldValidate === true) {
         if (field.fieldType === 'Float' || field.fieldType === 'Double' || field.fieldType === 'BigDecimal') {
             validators.push('@DecimalMin(value = "' + field.fieldValidateRulesMin + '")');
         } else {
+            const isLong = (field.fieldValidateRulesMin > MAX_VALUE || field.fieldType == 'Long') ? 'L' : '';
```

Comment:
No I meant I dont find any usage for `isLong` within this else block. Dont you have to append it to line below